OpenRA.Mods.Common/Traits/Buildings/FootprintUtils.cs

```diff
@@ -43,13 +43,20 @@ public static IEnumerable<CPos> FrozenUnderFogTiles(Actor a)
 			return Tiles(a.World.Map.Rules, a.Info.Name, a.Info.TraitInfo<BuildingInfo>(), a.Location, true);
 		}
 
-		public static IEnumerable<CPos> UnpathableTiles(string name, BuildingInfo buildingInfo, CPos position)
+		public static IEnumerable<CPos> TargetableTiles(string name, BuildingInfo buildingInfo, CPos position)
 		{
 			var footprint = buildingInfo.Footprint.Where(x => !char.IsWhiteSpace(x)).ToArray();
 			foreach (var tile in TilesWhere(name, buildingInfo.Dimensions, footprint, a => a == 'x'))
 				yield return tile + position;
 		}
 
+		public static IEnumerable<CPos> UnpathableTiles(string name, BuildingInfo buildingInfo, CPos position)
+		{
+			var footprint = buildingInfo.Footprint.Where(x => !char.IsWhiteSpace(x)).ToArray();
```

Comment:
Bibs have been an endless PITA.  My preferred solution would be to make `Bib` a visual-only trait (`WithBuildingBib` in `Mods.Cnc`?) and require mods to define non-mini bibs as part of the footprint using your new `u` tiles.  We then just need a `WVec` offset for the `CenterPosition` to avoid moving it down by half a cell.